docs/operator-manual/user-management/microsoft.md

```diff
@@ -147,7 +179,7 @@ policy.default: role:readonly
             p, role:org-admin, repositories, delete, *, allow
             g, "84ce98d1-e359-4f3b-85af-985b458de3c6", role:org-admin
 
-6. Mapping role from jwt token to argo
+4. Mapping role from jwt token to argo
 
     If you want to map the roles from the jwt token to match the default roles (readonly and admin) then you must change the scope variable in the rbac-configmap.
```

Comment:
The "scope" variable is not shown on the example above. It is a new property that readers must add? Or it exists in another part of the configmap that is not shown in the example above?

Maybe include a snipper of the "rbac-configmap" with the value of "scopes" clearly shown? I think it would benefit readers.